pkgs/development/python-modules/mdp/default.nix

```diff
@@ -9,11 +15,27 @@ buildPythonPackage rec {
     sha256 = "ac52a652ccbaed1857ff1209862f03bf9b06d093b12606fb410787da3aa65a0e";
   };
 
-  checkInputs = [ pytest ];
   propagatedBuildInputs = [ future numpy ];
 
-  # Tests disabled because of missing dependencies not in nix
-  doCheck = false;
+  checkInputs = [ pytest ];
+
+  doCheck = true;
+
+  pythonImportsCheck = [ "mdp" "bimdp" ];
+
+  postPatch = ''
+    # https://github.com/mdp-toolkit/mdp-toolkit/issues/92
+    substituteInPlace mdp/utils/routines.py --replace numx.typeDict numx.sctypeDict
+  '';
+
+  checkPhase = ''
+    runHook preCheck
+
+    pytest --seed 7710873 mdp
+    pytest --seed 7710873 bimdp
+
+    runHook postCheck
+  '';
```

Comment:
Can we use pytestCheckHook instead?

Reply:
No, it fails if the hook is used. We need to set the `seed` option to run the tests, and because of that each directory must be run separately, or it will complain that the option is already set.